countryspec/client.go

```diff
@@ -14,12 +14,12 @@ type Client struct {
 	Key string
 }
 
-// Get returns a CountrySpec for a given country code
-// For more details see https://stripe.com/docs/api/ruby#retrieve_country_spec
+// Get returns a CountrySpec for a given country code.
```

Comment:
Do you think we should expand this to "country specification" or "Country Spec"? It's not a big deal because we don't really seem to have a good name for what these actually are, but I noticed the API ref docs refer to it as the "Country Spec API".